boards/ikea-tradfri/include/board.h

```diff
@@ -31,10 +31,14 @@ extern "C" {
 /**
  * @name    Xtimer configuration
  *
- * The timer runs at 250 KHz to increase accuracy.
+ * The timer runs at 250 kHz to increase accuracy or 32768 Hz for LETIMER.
  * @{
  */
+#ifdef EFM32_USE_LETIMER
```

Comment:
This check is strange, since it uses `#Ifdef` if 0 or 1 this statement will evaluate as true.

Reply:
Will this be enough to fix the timer on `slstk3402a`?

Reply:
hmm no, something else must be off..